crates/biome_css_semantic/src/semantic_model/builder.rs

```diff
@@ -173,3 +185,23 @@ impl SemanticModelBuilder {
         }
     }
 }
+
+fn resolve_selector(current: &str, parent: &[Selector]) -> Vec<String> {
```

Comment:
It's possible that I am late to the party, so I am sorry if this questions come too late.

Is there a particular reason we need to track `String` inside the semantic model? I wonder if we can avoid that and store only `TextRange` and syntax nodes (syntax nodes are cheaper than strings) inside the model.